readthedocs/api/v3/views.py

```diff
@@ -110,7 +125,7 @@ class APIv3Settings:
     LimitOffsetPagination.default_limit = 10
 
     renderer_classes = (AlphabeticalSortedJSONRenderer, BrowsableAPIRenderer)
-    throttle_classes = (UserRateThrottle, AnonRateThrottle)
+    # throttle_classes = (UserRateThrottle, AnonRateThrottle)
```

Comment:
Why did we remove this?

Reply:
I temporary commented this because I was blocked immediately since we have the throttle very low in our settings.

https://github.com/readthedocs/readthedocs.org/blob/e70cf31d0f6302cc1d99d4c872683f7c96b6c5c8/readthedocs/settings/base.py#L902-L905

We should increase these values to be able to use APIv3 in the way we want with addons.